spec.emu

```diff
@@ -198,9 +198,10 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
           1. <ins>For each ModuleRequest Record _mr_ of _additionalRequests_, do</ins>
             1. <ins>Let _found_ be *false*.</ins>
             1. <ins>For each ModuleRequest Record _mr2_ of _requests_, do</ins>
-              1. <ins>If _mr_.[[Specifer]] is _mr2_.[[Specifer]] and _mr_.[[Phase]] is _mr2_.[[Phase]], then</ins>
+              1. <ins>If _mr_.[[Specifer]] is _mr2_.[[Specifer]], then</ins>
                 1. <ins>Assert: _found_ is *false*.</ins>
                 1. <ins>Set _found_ to *true*.</ins>
+                1. <ins>If _mr2_.[[Phase]] is ~source~ and _mr_.[[Phase]] is ~evaluation~, set _mr2_.[[Phase]] to ~evaluation~.</ins>
```

Comment:
Now that we use `GetSourceObject()` instead of an external slot, we can again deduplicate module requests and only ask to the host "I will need the module up to this phase". The host doesn't need the separate ModuleRequests to know when to create the source object anymore.

<sup>I still think that this was never necessary since it was never observable, but it's even less necessary now :P</sup>